Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1708,6 +1708,10 @@ extension BrowserViewController: IntroViewControllerDelegate {
     }
 
     func introViewControllerDidFinish(_ introViewController: IntroViewController) {
+        if self.profile.prefs.intForKey(PrefsKeys.IntroSeen) == nil, let url = URL(string: Strings.WhatsNewWebsite) {
+            let newTab = self.tabManager.addTab(URLRequest(url: url))
+            self.tabManager.selectTab(newTab)
+        }
```

Comment:
Can you move this into `viewDidAppear` instead? I think it's better if the "What's new" functionality is decoupled from the intro functionality.